lib/afinter.c

```diff
@@ -635,3 +641,107 @@ afinter_get_metrics(void)
 {
   return metrics;
 }
+
+/****************************************************************************
+ * Live collection of internal log messages.
+ ****************************************************************************/
+
+AFInterLive
+afinter_start_live_collection(void)
+{
+  AFInterSource *self;
+
+  if (is_live_collection)
```

Comment:
for a better concurrency handling this whole initialization must also be protected via `internal_msg_lock` mutex

Reply:
@HofiOne : okay, done 👍